connection_testcases.json

```diff
@@ -2557,6 +2557,33 @@
       "waitUntilAvailable": "PT30S"
     }
   },
+  {
+    "fs": {
+      "files": {
+        "/home/edgedb/test/credentials.json": "{}"
+      }
+    },
+    "name": "credentials_file_test_empty_credentials_file",
+    "opts": {
+      "credentialsFile": "/home/edgedb/test/credentials.json"
+    },
+    "result": {
+      "address": [
+        "localhost",
+        5656
+      ],
+      "branch": "__default__",
+      "database": "edgedb",
+      "password": null,
+      "secretKey": null,
+      "serverSettings": {},
+      "tlsCAData": null,
+      "tlsSecurity": "strict",
+      "tlsServerName": null,
+      "user": "edgedb",
```

Comment:
Shouldn't this be `admin`?

Reply:
Also the database should not be `edgedb` no?

Reply:
> Shouldn't this be admin? (or do we do some kind of edgedb -> admin conversion based on server version or something?)

We currently do the automatic edgedb->admin translation in the server. I'd like to set this to admin when using gel:// but that's not done yet -- all default users are edgedb for now.

> Also the database should not be edgedb no?

Unless I missed something, because it is unspecified, it's `branch = __default__`/`database = edgedb`.

Effectively the empty credentials file is equivalent to parsing `gel://`.

Reply:
Funny enough we don't have a test for the empty DSN only, but there are some reasonably close tests that resolve the same way as this empty credentials file does.